Makefile

```diff
@@ -101,15 +104,18 @@ endif
 
 # Run go vet against code
 vet:
-	$(GO) vet ./...
+	$(GOENV) CGO_ENABLED=1 go vet ./...
 
 tidy:
 	@echo "go mod tidy"
 	GO111MODULE=on go mod tidy
 	git diff --quiet go.mod go.sum
 
 image:
+	mkdir -p images/chaos-daemon/src
```

Comment:
Should we use a standalone Dockerfile at root to build all binaries? This can also forbid the leak of developer's information (such as path in log :sweat_smile: )

Copy source file is quite ugly :(

Reply:
Agree!

Reply:
Will you use a standalone Dockerfile in this pr?

Reply:
Yes! I have fixed this by build all binary in docker.

A Dockerfile and .dockerignore has been written in / to build binaries.